array list/array list/numerator.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Text;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace array_list
+{
+    public class CustomList<T> : IEnumerable<T>
```

Comment:
Looking for the other method's implementations in your CustomList (from the requirements)